test/night_in_salem_web/channels/game_session_channel_channel_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule NightInSalemWeb.GameSessionChannelChannelTest do
```

Comment:
Is this an intended test file? Looks like Channel is repeated twice in the filename and this Module name, and the tests are exactly the same as the other one with the proper name, unless I'm missing something.